test/Microsoft.AspNetCore.Server.Kestrel.FunctionalTests/Microsoft.AspNetCore.Server.Kestrel.FunctionalTests.csproj

```diff
@@ -8,6 +8,7 @@
     <PlatformTarget Condition="'$(TargetFramework)' == 'net46'">x64</PlatformTarget>
     <AutoGenerateBindingRedirects>true</AutoGenerateBindingRedirects>
     <GenerateBindingRedirectsOutputType>true</GenerateBindingRedirectsOutputType>
+    <ServerGarbageCollection>true</ServerGarbageCollection>
```

Comment:
Now that tests are fixed, I'll try and measure it. I set this so that our functional tests use the same GC setting Kestrel would normally run under.